src/Pages/SingleArticle.jsx

```diff
@@ -57,7 +59,10 @@ return (
               />
             ); 
         })}
-    </section>
+        
+            <AddNewComment article={singleArticle} topics={topics} setComments={setComments}/>
```

Comment:
Should this be above comments? It's much nicer to see the box at the top :)

Reply:
Done! Just working on the order of the comments!